sample/petstore-vertx-rx-server/src/main/java/io/swagger/server/api/MainApiVerticle.java

```diff
@@ -0,0 +1,75 @@
+package io.swagger.server.api;
+
+import java.nio.charset.Charset;
+
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.github.phiz71.vertx.swagger.router.OperationIdServiceIdResolver;
+import com.github.phiz71.vertx.swagger.router.SwaggerRouter;
+
+import io.swagger.models.Swagger;
+import io.swagger.parser.SwaggerParser;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Future;
+import io.vertx.core.file.FileSystem;
+import io.vertx.core.json.Json;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.Router;
+
+public class MainApiVerticle extends AbstractVerticle {

```

Comment:
Not related to this PR, but I wonder if we should make the generation of MainApiVerticle optional.

In my use case, I want to do dependency injection on verticles. I do that by scanning for classes of type verticle, then instantiating them in a spring context (so dependencies can be injected) and then deploy those existing verticle instances into vert.x. That does not work well with MainApiVerticle, and I would have to exclude it when scanning.

Not sure if my way of dealing with dependency injection in vert.x is common, though.